src/web/mjs/connectors/Kuimh.mjs

```diff
@@ -0,0 +1,68 @@
+import Connector from '../engine/Connector.mjs';
+import Manga from '../engine/Manga.mjs';
+
+export default class Kuimh extends Connector {
+    constructor() {
+        super();
+        super.id = 'kuimh';
+        super.label = '酷爱漫画 (Kuimh)';
+        this.tags = [ 'manga', 'webtoon', 'chinese' ];
+        this.url = 'https://www.kuimh.com';
+
+        this.path = '/booklist?page=';
+        this.pathMatch = 'page=(\\d+)';
```

Comment:
May prefer regex literal instead of string
`/page=(\d+)/`

Reply:
i tried `/page=(\d+)/` and '/page=(\\d+)/' before but it will give error `Cannot read property '1' of null`

Reply:
Thats weird, may check this out next time some work is needed in this connector...